wazo_dird/plugins/phonebook_backend/schemas.py

```diff
@@ -26,3 +32,19 @@ class ListSchema(_ListSchema):
 source_list_schema = SourceSchema(many=True)
 source_schema = SourceSchema()
 list_schema = ListSchema()
+
+
+class CountParams(TypedDict):
+    search: str | None
+
+
+class ContactListSchema(_ListSchema):
+    searchable_columns = ['uuid', 'firstname', 'lastname']
+    sort_columns = ['firstname', 'lastname']
+    default_sort_column = 'lastname'
+
+    def load_count(self, args: dict, **kwargs) -> CountParams:
+        return cast(CountParams, projection(self.load(args, **kwargs), ['search']))
```

Comment:
IMHO this logic should stay contained in `http.py`. The reason is that if we change the counting logic, we should not have to change the schema code. That's more an opinion open to discussion though :)

If we keep this logic in here, we should rename it somewhat, because my first reaction was "what does count have to do with the schema???"